metadata/src/main/java/org/apache/kafka/controller/OffsetControlManager.java

```diff
@@ -71,7 +71,7 @@ Builder setTime(Time time) {
             return this;
         }
 
-        public OffsetControlManager build() {
+        OffsetControlManager build() {
```

Comment:
nit: The Builder itself can also be package-local.

Reply:
Done